conda_index/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+conda index. Create repodata.json for collections of conda packages.
+"""
+
+__version__ = "0.0.1"
```

Comment:
We settled on CalVer for conda-libmamba-solver since any more complicated, "semantic" meaning for versions is not clear for users and doesn't really matter. Each release would basically make clear which version of conda-build it's compatible with, in the conda package metadata.